model/blocks/blockgraph.go

```diff
@@ -0,0 +1,37 @@
+package blocks
+
+import (
+	"fmt"
+	"hash/crc32"
+	"strings"
+)
+
+type BlockNode struct {
+	Block        string
+	Height       uint64
+	Parent       string
+	ParentHeight uint64
+	Miner        string
+}
+
+var defaultTbl = crc32.MakeTable(crc32.Castagnoli)
+
+func (b *BlockNode) DotString() string {
```

Comment:
nit: this can just be `String()`

Reply:
Disagree. This is not for human readability like String intends and is specific to Dot. This seems like the best name while leaving room for String later. That said, I made this a private method anyway, so this probably matters a whole lot less now.